iati/core/tests/test_data.py

```diff
@@ -462,6 +464,7 @@ def test_detect_version_implicit_parent_matches_implicit_child(self, iati_tag_na
         assert result == '1.01'
 
     def test_detect_version_explicit_parent_matches_implicit_child(self, iati_tag_names):
+        """Check that the default version is detected for a dataset with the default version explicitly defined at `iati-activities` level, but where all `iati-activity` child elements are not defined (i.e. the default version is assumed)."""
         data = iati.core.Dataset("""
         <{0} version='1.01'>
```

Comment:
Using single quotes for attributes doesn't seem right. Suggest using double quotes.

(the same for a couple of others like this)